js/scripts/recent-works.js

```diff
@@ -0,0 +1,165 @@
+const hamburger = document.querySelector('#mob-menu');
+const navBar = document.querySelector('.mob-nav-list');
+const closeBtn = document.querySelector('.close-button');
+const navItems = document.querySelectorAll('.nav-item');
+let isNavVisible = false;
+
+var projects = [
```

Comment:
- [ ] I understand you need this variable `projects` globally that's why you used var here, But I can not approve the project with linter errors, Please fix this issue.
- To fix this you can use const here and export this variable to your second javascript file but for that, you also need to use these javascript files as modules. Please read this article [javascript export import](https://javascript.info/import-export)